src/lib/ngsi/Scope.h

```diff
@@ -31,7 +31,7 @@
 #include "common/Format.h"
 #include "orionTypes/areas.h"
 
-using namespace orion;
+//using namespace orion;
```

Comment:
+1

Yeah, if in a header file it gets spread to *every* source file that somehow includes the header file.
Seems like a very bad idea.
Remove the comment (I mean, the entire line :-)) ...